utils/src/main/java/org/jboss/windup/util/config/ConfigurationFiles.java

```diff
@@ -0,0 +1,64 @@
+package org.jboss.windup.util.config;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jboss.forge.furnace.util.Visitor;
+import org.jboss.windup.util.exception.WindupException;
+
+/**
+ * Utility for loading configuration files matching a specific filename suffix pattern.
+ * 
+ * @author <a href="mailto:lincolnbaxter@gmail.com">Lincoln Baxter, III</a>
+ */
+public class ConfigurationFiles
```

Comment:
Nitpicking, but I don't love the name. I don't see anything in here specific to configuration files? It just looks like a nice file-path scanner utility to me.